XamlControlsGallery/ControlPages/CornerRadiusPage.xaml

```diff
@@ -0,0 +1,67 @@
+<Page
+    x:Class="AppUIBasics.ControlPages.CornerRadiusPage"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:Windows10version1809="http://schemas.microsoft.com/winfx/2006/xaml/presentation?IsApiContractPresent(Windows.Foundation.UniversalApiContract, 7)"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:local="using:AppUIBasics"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    Background="{ThemeResource ApplicationPageBackgroundThemeBrush}"
+    mc:Ignorable="d">
+
+    <StackPanel>
+        <TextBlock IsTextSelectionEnabled="True">
+            <LineBreak />
+            <Run Text="For UI elements like basic controls like buttons that users see on screen at all times," />
+            <Run Text="the default radius value we use for these UI elements is" />
+            <Run FontWeight="bold" Text="2px" />
+            <Run Text="." />
+            <LineBreak />
+            <LineBreak />
+            <Run Text="For transient UI elements that appear on screen temporarily, like MenuFlyout, or elements that overlay other UI, like TabView tabs," />
+            <LineBreak />
+            <Run Text="the default radius value we use for these UI elements is" />
+            <Run FontWeight="Bold" Text="4px" />
+            <Run Text="." />
+        </TextBlock>
+        <local:ControlExample
+            x:Name="Example1"
+            HeaderText="A button with CornerRadius"
+            MinimumUniversalAPIContract="1809">
```

Comment:
Instead of setting the minimum contract, we should probably show how to use conditional namespaces and add a disclaimer at the top of the page.

Reply:
We will need both. The reason this is causing the red warning text is because this value should be "7" (which is the version of the API contract this first appeared in) and not "1809" which is the device family release. https://docs.microsoft.com/en-us/uwp/api/windows.ui.xaml.controls.control.cornerradius?view=winrt-19041